hibernate-reactive-core/src/main/java/org/hibernate/reactive/persister/collection/impl/ReactiveOneToManyPersister.java

```diff
@@ -106,7 +106,7 @@ private ReactiveConnection getReactiveConnection(SharedSessionContractImplemento
 
 			return reactiveConnection.update( sql, params.toArray( new Object[0] ) );
 		}
-		return CompletionStages.completedFuture( -1 );
+		return CompletionStages.completedFuture( 0  );
```

Comment:
Shouldn't this return `CompletionStage<Void>`?

Reply:
only reason is the ```reactiveConnection.update( sql, params.toArray( new Object[0] ) );``` returns ```CompletionStage<Integer>```

Guess we could convert to Void?

Reply:
OK, but all the other methods are Void, why this should be different?
I think I mentioned this inconsistency before. The method in `ReactiveCollectionPersister` should return `CompletionStage<Void>`: https://github.com/hibernate/hibernate-reactive/blob/384eb90a3a7e24f6e2b14e87d8d98689e4137075/hibernate-reactive-core/src/main/java/org/hibernate/reactive/persister/collection/impl/ReactiveCollectionPersister.java#L31